client/src/js/base/Modal.js

```diff
@@ -10,3 +15,135 @@ export const Modal = styled(BsModal)`
         margin: -1px 0 0;
     }
 `;
+
+const modalOverlayIn = keyframes`
+    0% {
+        background: rgba(255, 255, 255, 0);
+    }
+    100% {
+        background: rgba(0, 0, 0, 0.5);
+    }
+`;
+
+const modalOverlayOut = keyframes`
+    0% {
+      background: rgba(0, 0, 0, 0.5);
+    }
+    100% {
+        background: rgba(255, 255, 255, 0);
+    }
+`;
+
+const modalContentIn = keyframes`
+    0% {
+      opacity: 0;
+    }
+    100% {
+      transform: translate(0,80px);
+    }
+`;
+
+const modalContentOut = keyframes`
+    0% {
+      transform: translate(0,80px);
+    }
+    100% {
+        opacity: 0;
+    }
+`;
+export const ModalDialogOverlay = styled(({ show, ...rest }) => <DialogOverlay {...rest} />)`
+    z-index: 9999;
+
+    animation: ${props => (props.out ? modalOverlayOut : modalOverlayIn)} 0.4s;
+    animation-fill-mode: forwards;
+`;
+
+export const ModalDialogContent = styled(({ size, ...rest }) => <DialogContent {...rest} />)`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['size' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4961580)